src/include/mpir_sched.h

```diff
@@ -0,0 +1,15 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *  (C) 2019 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ *
+ */
+
+#ifndef MPIR_SCHED_H_INCLUDED
+#define MPIR_SCHED_H_INCLUDED
+
+typedef struct MPIDU_Sched_state MPIR_Sched_list_t;
```

Comment:
The `MPIDU_` functions/datatypes should have similar names as the `MPIR_` versions.